### PR TITLE
Set up index.html

### PR DESCRIPTION
**Pull request by chloesw**
Signed-off-by: swzoey <tmddnnim2da@gmail.com>

### CODE REVIEW

File: research/frontend/css/index.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+html, body {
+    width: 100%;
+    margin: 0 auto;
+    background-color: #f7f7f7;
+    color: black;
+  }
```
Comment (harrydrippin) [inline research/frontend/css/index.css]: Take care of your code format. No spaces required here.

File: research/frontend/css/index.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+nav {
+  height: 80px;
+}
+
+.qc-logo-icon {
+    width: auto;
+    height: 40px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
```
Comment (harrydrippin) [inline research/frontend/css/index.css]: Same as above.

File: research/frontend/js/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+$(document).ready(function () {
+    $(".modal").modal();
+  
+    $("#edit-interests").click(function (e) {
+      e.preventDefault();
```
Comment (harrydrippin) [inline research/frontend/js/index.js:5]: Tab size should be fixed to exact one size. This code uses both 2 and 4 spaces. I recommend fixing your editor config `tabSize` to **4** on Javascript.

**Review by harrydrippin — COMMENTED** [review]
Please take a look.

**Comment by ethkim** [comment]
Thanks! 

Is there a way to test this? Do we have a screenshot based on the code you wrote here?
Eom will test it! @royeom

File: research/frontend/css/index.css (hunk 1 above)
Comment (chloesw) [reply research/frontend/css/index.css]: okay, I'll check it

File: research/frontend/js/index.js (hunk 3 above)
Comment (chloesw) [reply research/frontend/js/index.js:5]: okay, i'll try to fix and apply it.

File: research/frontend/vue.js/index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        el: '#app-2',
+        data: {
+            message: '이 페이지는 ' + new Date() + '에 로드 되었습니다'
+        }
+    })
+
+    var app4 = new Vue({
+        el:'#app-4',
+        data: {
+            todos: [ //배열의 데이터를 사용해 항목 목록을 표시하는데 사용할 수 있다.
```
Comment (ethkim) [inline research/frontend/vue.js/index.html]: Make all code + comment in English please. :)

**Comment by royeom** [comment]
@ethkim 
Now, We are trying to test .
I will try as soon as possible !

File: research/frontend/js/login.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+function auth(userName, password) {
+    $.ajax
+        ({
+            type: "POST",
+            //SEND TO MY SERVER URL
+            url: "eomserver",
+            dataType: 'json',
+            async: false,
+            data: '{"userName": "' + userName + '", "password" : "' + password + '"}',
```
Comment (royeom) [inline research/frontend/js/login.js:19]: You have to change key from user'N'ame to user'n'ame !

**Review by royeom — COMMENTED** [review]
Please take a look.

File: research/frontend/js/login.js (hunk 5 above)
Comment (chloesw) [reply research/frontend/js/login.js:19]: okay!

**Pull request merged**